src/Client/App.fs

```diff
@@ -22,17 +23,15 @@ type PageModel =
     | WishListModel of WishList.Model
 
 type Msg =
-    | LoggedIn
+    | LoggedIn of UserData
     | LoggedOut
     | StorageFailure of exn
-    | OpenLogIn
-    | MenuMsg of Menu.Msg
     | LoginMsg of Login.Msg
     | WishListMsg of WishList.Msg
-    | Logout
+    | Logout of unit
```

Comment:
What is the point of that `unit`?

Reply:
Is it because you wanted to avoid `Menu.view (fun () -> dispatch Logout) model.User` down below?

Reply:
Exactly, it's to be able to just `Menu.view (Logout >> dispatch) model.User`.

Same thing could be applied to `ClickLogIn` on login page. The usage would be `OnClick (ignore >> ClickLogIn >> dispatch)`.